.gitignore

```diff
@@ -3,7 +3,6 @@
 
 # This is dynamic in a monorepo
 bin/internal/engine.version
-bin/internal/engine.realm
```

Comment:
Would this leave user's checkouts dirty?

Reply:
Only if the user has FLUTTER_REALM set.

Reply:
oh wait, nvm, i guess we'd only write this on CI?